src/test/java/br/com/matheus/agendamentoconsultas/base/AbstractValidacaoAgendamentoConsultaBaseTest.java

```diff
@@ -31,9 +32,11 @@ public abstract class AbstractValidacaoAgendamentoConsultaBaseTest extends Mocke
     @BeforeAll
     static void setUpBeforeAll() {
         medicoId = 1L;
+        medico = new Medico();
+        setPrivateId(medico, medicoId);
```

Comment:
Tem um recurso de mock, que daria pra usar e evitar reflection

```suggestion
        medico = spy(new Medico());
        doReturn(medicoId).when(medico).getId();
```

Depois da uma pesquisada pra voce entender o conceito

Reply:
@bugorin Não havia me atentado, mas realmente o mock caberia melhor nessa situação. Removi a refection utilizada e mockei o retorno dos ids.